src/AutoRest.CSharp/Mgmt/AutoRest/MgmtOutputLibrary.cs

```diff
@@ -139,15 +140,15 @@ private IEnumerable<InputType> UpdateBodyParameters()
                         if (parameter.Location != RequestLocation.Body)
                             continue;
 
-                        IncrementCount(usageCounts, parameter.Type.Name);
+                        IncrementCount(usageCounts, parameter.Type.GetImplementType().Name);
```

Comment:
OK so if we have this method, we have so many other places doing
```csharp
type = type is InputNullableType ? type.Type : type;
```
all of them should be changed to use this `GetImplementType` method.